templates/website/index-index.html

```diff
@@ -6,6 +6,10 @@
 
 <div class="row row-full-width hero">
     <div class="large-10 large-centered columns">
+        <a href="https://www.mysociety.org/about/anniversary" id="mysociety-badge" aria-label="Visit mySociety 20 years anniversary">
```

Comment:
I was about to say we don’t need the `aria-label` here as [assistive devices will use the `alt` text of the child image automatically](https://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/H30.html) … but since there are _two_ images in this link, I’m not sure whether that’s true! Still, I’d probably go for a shorter `aria-label`, without the "visit" prefix (it’s already a link, no need to tell people that they’ll be "visiting" the destination, just tell them what the destination is). "20 years of mySociety" or "mySociety’s 20th anniversary" is probably fine.